### ISSUE TITLE
Loss_l : inf in training ?

### ISSUE THREAD

**Issue by transcendentsky** [ruinmessi/RFBNet#35]
Hello, first of all, thanks for your code releasing.
I got the training loss inf, acutally loss_l = inf, i use your original code (only fixed some bug), but i don't know why i got inf.
Parameters: lr:0.004, batchsize:32, base_model:vgg_reducedfc.pth
GPU: 1080ti

Any comments will be appreciated.
Thanks very much!

**Comment by ruinmessi** [comment]
@transcendentsky  You may look at this [pull](https://github.com/amdegroot/ssd.pytorch/pull/116) for details.

**Comment by transcendentsky** [comment]
I have tried this way, but the problem still exists.

**Comment by transcendentsky** [comment]
I tracked the code and find the problem comes to prior (prior_box) , the code `s_k = self.min_sizes[k]/self.image_size` get 0 because these are two integers. I compared it with code in  `https://github.com/amdegroot/ssd.pytorch` , and There should have a `from __future__ import division` on the top and code goes right.

**Comment by ruinmessi** [comment]
@transcendentsky  The current code has no been tested in Python2.
